docs/commands/secrets.md

```diff
@@ -187,56 +187,71 @@ To ensure that your command’s arguments and options are passed correctly you m
 $ torus run -e production -s www -- node ./bin/www --app api
 ```
 
-## ls
-###### Added [v0.13.0](https://github.com/manifoldco/torus-cli/blob/master/CHANGELOG.md)
-
-`torus ls [path]` allows a user to explore the objects and values set within their organization.
+## list
+###### Added [v0.28.0](https://github.com/manifoldco/torus-cli/blob/master/CHANGELOG.md)
 
-Each level within the organization can be inspected by changing the segments supplied in the path. Wildcards cannot be used for the organization or project segments of the [path](../concepts/path.md).
+`torus list [name]...` allows a user to explore the objects and values set within their organization. Users can find the location of a particular secret by passing its name as an argument, or view all secrets within different branches of their organization.
 
-Path is required, and does not support context.
+Each level within the organization can be inspected by changing the options supplied to the command.
 
 ### Command Options
 
   Option | Description
   ---- | ----
+  --org, -o | Required flag to specify org.
+  --project, -p | Required flag to specify project.
+  --env, -e | Specify environment filter(s) for displayed secrets. To specify multiple environments, pass multiple flags (eg.`torus list -e env1 -e env2`). This flag is optional.
+  --service, -s | Specify service filter(s) for displayed secrets. To specify multiple services, pass multiple flags (eg. `torus list -s ser1 -s ser2`). This flag is optional.
   --verbose, -v | Show which type of path is being displayed, shortcut for --format=verbose
```

Comment:
also, there is no format flag!

Reply:
Good catch, second time I've missed a ref to that.